test/object-store/test_runner.cpp

```diff
@@ -138,6 +138,7 @@ class EvergreenReporter : public CumulativeReporterBase {
     }
     void testCaseStarting(TestCaseInfo const& testCaseInfo) override
     {
+        std::cerr << "Starting test case \"" << testCaseInfo.name << "\"\n";
```

Comment:
Nice! - I had wondered how we could add this sort of labeling around test cases and sections
What do you think about a slightly different format so the text stands out amongst the sea of text. For example:
```
>> START TEST CASE: "C API - client reset"
. . .
>> START SECTION: "C API - client reset::Manual reset"
. . .
<< END SECTION: "C API - client reset::Manual reset"
. . .
<< END TEST CASE: "C API - client reset"
```